service.py

```diff
@@ -36,11 +34,15 @@ def handler(event, context):
     return results
 
 def parseRecords(records):
+    """Simple method to parse list of records and process each entry."""
     logger.debug("Parsing Messages")
     results = list(map(parseRecord, records))
     return results
```

Comment:
Just a style thing, but I'm a fan of fewer variables, i.e. `return list(map(parseRecord, records))` (but see comment above about maybe getting rid of this function.

Also, I would have expected a list comprehension like ` return [parseRecord(r) for r in records]`. There's not much of a _technical_ difference -- but when in Rome...

Reply:
No, but I switched it to list comprehension. I'm not sure why I started using `map()` but I was experimenting with it in an earlier version of the boilerplate code this is based on and its a carry-over here.